agent/rpc/middleware/interceptors.go

```diff
@@ -19,9 +20,16 @@ import (
 // used for this historically is "RPC", so we continue to use that here.
 const RPCTypeInternal = "internal"
 
-var metricRPCRequest = []string{"rpc", "server", "request"}
+var metricRPCRequest = []string{"rpc", "server", "call"}
 var requestLogName = "rpc.server.request"
 
+var NewRPCCounters = []prometheus.CounterDefinition{
+	{
+		Name: metricRPCRequest,
```

Comment:
any reason to not in-line here? Do we use `metricRPCRequest` anywhere else?

Reply:
looks like the others in `agent/setup.go` inline as a `[]string` too, so would be consistent with that at the very least 🤷🏼

Reply:
I was using this a bunch for some testing here: https://github.com/hashicorp/consul/pull/12585 and in the `Record()` call below

🙈 Unsure what the most gopher way to do this is but I am willing to punt and take another pass over this as needed!